### PR TITLE
fix a log param error

### PR DESCRIPTION
**Pull request by WanLinghao**
**What this PR does / why we need it**:
As the patch shows, it fix a small log param error in pkg/kubectl/cmd/get/get.go
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by WanLinghao** [comment]
@soltysh PTAL

File: pkg/kubectl/cmd/get/get.go [hunk 1]
```diff
@@ -591,7 +591,7 @@ func (o *GetOptions) watch(f cmdutil.Factory, cmd *cobra.Command, args []string)
 func attemptToConvertToInternal(obj runtime.Object, converter runtime.ObjectConvertor, targetVersion schema.GroupVersion) runtime.Object {
 	internalObject, err := converter.ConvertToVersion(obj, targetVersion)
 	if err != nil {
-		glog.V(1).Infof("Unable to convert %T to %v: err", obj, targetVersion, err)
+		glog.V(1).Infof("Unable to convert %T to %v: %v", obj, targetVersion, err)
```
Comment (nikhita) [inline pkg/kubectl/cmd/get/get.go]: @WanLinghao thanks for catching this!

**Comment by nikhita** [comment]
/assign @soltysh 
for approval

**Comment by WanLinghao** [comment]
@soltysh  PTAL thanks

**Comment by WanLinghao** [comment]
@soltysh  friendly ping

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Pull request merged**